lib/deps/binary/base64.js

```diff
@@ -4,10 +4,10 @@ var buffer = require('./buffer');
 
 exports.atob = function (str) {
   var base64 = new buffer(str, 'base64');
-  // Node.js will just skip the characters it can't encode instead of
-  // throwing and exception
+  // Node.js will just skip the characters it can't decode instead of
+  // throwing an exception
   if (base64.toString('base64') !== str) {
-    throw ("Cannot base64 encode full string");
+    throw 'InvalidBase64Error';
```

Comment:
This still gets discarded immediately, so I'm not sure what error to throw... Maybe just `throw new Error();`?